feat: Add a fact about dreaming. (#16)
octopusthink/funfactsaboutoctopus.es: src/content/facts/11-dreams.md

---
title: They (probably) dream, and it's awfully colourful.
source: Science Alert
sourceUrl: https://www.sciencealert.com/watch-the-mesmerising-colour-shifts-of-a-sleeping-octopus
image: /images/Haeckel_Hexacoralla.jpg
imageSource: https://www.oldbookillustrations.com/illustrations/octopus/
---

> Do cephalopods dream? Scientists aren't sure, but there's some evidence indicating that they experience a phase of sleep similar to REM sleep in humans. Sleeping octopuses actually flicker, changing colours and patterns like some kind of multi-armed lava lamp. 

Reviewer: ```suggestion
Do cephalopods dream? Scientists aren't sure, but there's some evidence indicating that they experience a phase of sleep similar to REM sleep in humans. Sleeping octopuses actually flicker, changing colours and patterns like a multi-armed lava lamp. 
```

> These colour and pattern changes are similar to the way that octopuses use colour changes to camoflague themselves whilst awake. This has led scientists to theorise that octopuses may be dreaming about eating tasty crab or running away from a shark.

Reviewer: ```suggestion
These colour and pattern changes are similar to the way that octopuses camoflague themselves whilst awake, leading scientists to theorise that octopuses may be dreaming about eating tasty crab. (Or running away from a shark!)
```
